tests/scripts/helm.sh

```diff
@@ -34,8 +34,8 @@ install() {
         # shellcheck disable=SC2021
         dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
         mkdir -p "${temp}"
-        wget "https://storage.googleapis.com/kubernetes-helm/helm-v2.6.0-${dist}-${arch}.tar.gz" -O "${temp}/helm-v2.6.0-${dist}-${arch}.tar.gz"
-        tar -C "${temp}" -zxvf "${temp}/helm-v2.6.0-${dist}-${arch}.tar.gz"
+        wget "https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-${dist}-${arch}.tar.gz" -O "${temp}/helm-v2.11.0-${dist}-${arch}.tar.gz"
```

Comment:
@galexrt This can be updated independent of your investigation for #2160, right?

Reply:
Sure, will revert this

Reply:
Seems like it's fine to update for the CI system, just wanted to confirm if there is any other issue with it. This alone shouldn't prevent users from installing with an older version of helm, right? That was just the question with #2160.

Reply:
Oh sorry I already reverted. I had an issue with the helm test and upgrading fixed it. Maybe it works on CI system but I encountered issues with this test on my env.
I don't think this would prevent installing with an older version of helm, no.
Should I keep it at 2.6?